wrapanapi/lenovo.py

```diff
@@ -3,10 +3,10 @@
 
 Used to communicate with providers without using CFME facilities
 """
-from base import WrapanapiAPIBase
+from base import WrapanapiAPIBaseVM
```

Comment:
Would not be WrapanapiAPIBase against WrapanapiAPIBaseVM?

Reply:
Yes
lemme fix that

Reply:
Done